app/src/main/res/drawable/thumb_werknemerloket.xml

```diff
@@ -0,0 +1,36 @@
2,-0.2 -0.7,-0.4 -1.1,-0.5c-0.4,-0.1 -0.9,-0.3 -1,-0.5c-0.2,-0.2 -0.6,-0.5 -0.9,-0.7c-0.7,-0.5 -0.9,-0.8 -1.3,-2.1C47.3,8.7 46.8,8.4 45.8,8.8M46.7,15c0.2,0.1 0.4,0.3 0.5,0.4c1.5,1.3 2.1,1.6 3.1,1.6c0.4,0 3,1.4 3.2,1.7c0.1,0.1 0.4,0.3 0.6,0.5c0.2,0.1 0.5,0.4 0.6,0.6s0.3,0.4 0.4,0.4c0.3,0 1.7,1.5 1.8,1.9c0,0.2 0.3,0.7 0.5,1.2c0.6,1.3 0.6,4.4 0.1,6c-0.2,0.5 -0.4,1.4 -0.5,1.9s-0.3,1.1 -0.4,1.2c-0.2,0.3 -0.1,0.8 0.4,1.8c0.4,0.8 0.5,1.4 0.3,2.5c-0.1,0.3 -0.2,0.9 -0.3,1.4c-0.1,0.4 -0.2,0.8 -0.3,0.9c-0.1,0.1 -0.2,0.2 -0.2,0.3s-0.1,0.2 -0.2,0.2c-0.1,0.1 -0.4,0.4 -0.6,0.7S55.1,40.9 55,41c-0.2,0.1 -0.3,0.3 -0.3,0.4c0,0.1 -0.2,0.6 -0.5,1c-0.3,0.5 -0.5,0.9 -0.5,1s-0.2,0.4 -0.4,0.6c-0.2,0.2 -0.6,0.8 -0.8,1.3s-0.5,0.9 -0.6,1c-0.2,0.1 -0.2,2.2 0.1,2.6c0.1,0.2 0.3,0.7 0.4,1.2c0.2,0.9 0.8,2.1 1,2.1c0.1,0 0.2,0.2 0.3,0.4c0.1,0.2 0.4,0.5 0.7,0.7c0.3,0.2 0.5,0.4 0.6,0.5c0,0.1 0.5,0.3 0.9,0.5c0.5,0.2 1.1,0.4 1.4,0.6s1.2,0.5 1.9,0.6c0.7,0.2 1.5,0.4 1.7,0.6c0.2,0.1 0.7,0.3 1,0.4c0.7,0.2 1.4,0.5 1.7,0.7c0.1,0.1 0.5,0.3 1,0.4c0.4,0.1 1.1,0.4 1.5,0.7c0.4,0.3 0.8,0.5 0.9,0.5s0.3,0.2 0.5,0.4c0.2,0.2 0.3,0.4 0.4,0.4c0.2,0 0.7,0.5 0.8,0.8c0,0.1 0.2,0.4 0.4,0.5c0.3,0.2 0.4,0.5 0.5,1c0.1,0.4 0.3,0.9 0.5,1.1c0.3,0.4 0.4,0.7 0.7,2.1c0.1,0.4 0.2,0.7 0.3,0.8c0.3,0.3 0.5,1.2 0.8,2.4c0.1,0.6 0.3,1.4 0.5,1.8c0.4,0.8 0.4,1 0.6,2.8c0.1,1 0.2,1.7 0.4,2c0.6,1.3 0.7,7.8 0.2,8.5c-0.2,0.2 -0.4,0.5 -0.5,0.8c-0.1,0.2 -0.3,0.5 -0.5,0.6s-0.5,0.3 -0.7,0.6c-0.3,0.3 -0.7,0.5 -1.2,0.6c-0.4,0.1 -0.9,0.2 -1,0.3c-0.3,0.3 -1.1,0.5 -2.1,0.7c-0.5,0.1 -1,0.2 -1.1,0.3c-0.4,0.3 -1.1,0.5 -2.2,0.7c-0.6,0.1 -1.4,0.3 -1.9,0.5c-0.6,0.3 -1.2,0.4 -2.6,0.6c-1,0.1 -2.2,0.3 -2.7,0.5c-2.6,0.8 -15.6,0.8 -19.1,0c-0.8,-0.2 -2.7,-0.4 -5,-0.7c-0.6,-0.1 -1.3,-0.2 -1.8,-0.4C30.6,88.3 30,88.1 29,88c-1.6,-0.2 -2.2,-0.3 -2.6,-0.6c-0.2,-0.2 -0.6,-0.3 -1.1,-0.3c-1.1,-0.1 -2.7,-0.5 -3,-0.8c-0.1,-0.1 -0.6,-0.2 -1,-0.3c-1,-0.2 -1.6,-0.4 -2,-0.8c-0.1,-0.1 -0.5,-0.3 -0.8,-0.4c-0.5,-0.2 -1.1,-0.7 -1.4,-1.3c-0.5,-1 -0.3,-7.6 0.4,-9.9c0.1,-0.4 0.3,-1.5 0.4,-2.6c0.2,-1.9 0.6,-4.1 0.8,-4.3c0.1,-0.1 0.2,-0.6 0.3,-1.2c0.2,-1.1 0.5,-1.7 0.8,-2c0.1,-0.1 0.2,-0.4 0.3,-0.7c0.2,-0.6 1,-1.8 1.5,-2c0.2,-0.1 0.4,-0.3 0.4,-0.4c0.1,-0.3 1.2,-1.3 1.8,-1.6c0.2,-0.1 0.5,-0.3 0.5,-0.4c0.1,-0.2 0.8,-0.6 1.4,-0.7c0.3,0 0.8,-0.3 1.1,-0.5s0.9,-0.5 1.3,-0.6c0.4,-0.1 1,-0.3 1.2,-0.5c0.3,-0.2 0.9,-0.4 1.4,-0.6c0.5,-0.1 1.1,-0.4 1.3,-0.5s0.9,-0.4 1.5,-0.6c0.7,-0.2 1.2,-0.4 1.3,-0.6c0.1,-0.1 0.4,-0.4 0.7,-0.5s0.5,-0.4 0.6,-0.5c0,-0.1 0.2,-0.4 0.4,-0.5c0.3,-0.2 1.3,-2.3 1.5,-3.3c0.3,-1.1 0.1,-3.1 -0.4,-3.7c-0.2,-0.3 -0.5,-0.8 -0.6,-1s-0.3,-0.7 -0.5,-1.1c-0.2,-0.4 -0.4,-0.9 -0.5,-1.3c-0.2,-0.7 -0.8,-1.7 -1.3,-2.2c-1.3,-1.2 -1.9,-3.9 -1.2,-5.2c0.5,-0.8 0.5,-1.8 0.2,-4.2c-0.5,-3.8 -0.2,-7.1 0.8,-8.7c0.3,-0.4 0.5,-0.8 0.5,-0.9c0,-0.1 0.5,-0.7 1.6,-1.9c0.6,-0.7 0.9,-0.9 2.1,-1.7c0.4,-0.2 0.7,-0.6 0.9,-0.7c0.1,-0.1 0.4,-0.3 0.6,-0.4c0.2,-0.1 0.5,-0.3 0.7,-0.4c0.6,-0.4 1.4,-0.4 1.9,-0.1l0.4,0.3l0.5,-0.3c0.3,-0.1 0.8,-0.3 1.1,-0.3s0.8,-0.1 1,-0.2C46.3,14.7 46.3,14.7 46.7,15"
```

Comment:
This is a very long vector path (6206 characters), which is bad for performance. Please consider removing minor details. I know this will be a tedious task. The only reason this logo is still not included with andOTP because I was hesitant of all the work.

Reply:
Hi, I don't know how to even tackle this, leaving this for the future